src/pepr/operator/controllers/monitoring/pod-monitor.ts

```diff
@@ -0,0 +1,98 @@
+import { V1OwnerReference } from "@kubernetes/client-node";
+import { K8s, Log } from "pepr";
+import { Monitor, PrometheusPodMonitor, UDSPackage } from "../../crd";
+import { getOwnerRef } from "../utils";
+import { generateMonitorName } from "./common";
+
+/**
+ * Generate a pod monitor for a pod
+ *
+ * @param pkg UDS Package
+ * @param namespace
+ */
+export async function podMonitor(pkg: UDSPackage, namespace: string) {
+  const pkgName = pkg.metadata!.name!;
+  const generation = (pkg.metadata?.generation ?? 0).toString();
+  const ownerRefs = getOwnerRef(pkg);
+
+  Log.debug(`Reconciling PodMonitors for ${pkgName}`);
```

Comment:
We just switched our logging approach to use our own logger rather than the default Pepr one. That PR might be a good place to see the diffs - https://github.com/defenseunicorns/uds-core/pull/533/files. For you the changes should just be import `Component` and `setupLogger` from our `logger` file (`import { Component, setupLogger } from "../../../logger";` should be right), then add `const log = setupLogger(Component. OPERATOR_MONITORING);`. And then just switch `Log` -> `log` and it should all work the same way.

I think in the other files where there were conflicts I resolved and switched to the new `log`, but might be good to validate.